src/pages/components/accordion/usage.mdx

```diff
@@ -4,20 +4,180 @@ description: The accordion component delivers large amounts of content in a smal
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
+<AnchorLink>Behavior</AnchorLink>
+<AnchorLink>Content</AnchorLink>
+<AnchorLink>Related</AnchorLink>
+<AnchorLink>References</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
+
+</AnchorLinks>
+
+## Overview
+
+The accordion component delivers large amounts of content in a small space through progressive disclosure. The header title give the user a high level overview of the content allowing the user to decide which sections to read.
+
+Accordions can make information processing and discovering more effective. However, it does hide content from users and its important to account for a user not noticing or reading all of the included content. If a user is likely to read all of the content then don't use an accordion as it adds the burden of an extra click, instead use a full scrolling page with normal headers.
+
+### When to use:
+
+- To organize related information.
+- To shorten pages and reduce scrolling when content is not crucial to read in full.
+- When space is at a premium and long content cannot be displayed all at once, like on a mobile interface or in a side panel.
+
+## Formatting
+
+### Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![accordion anatomy](images/accordion-anatomy.png)
+
+</Column>
+</Row>
+
+1. **Header**: contains the section title and is control for revealing the panel.
+2. **Icon**: indicates if the panel is open or closed
+3. **Panel**: the section of content associated with an accordion header.
+
+### Alignment
+
+By default the chevron icon is placed on the `end` side of the header. This allows for the title on the `start` side to align with other type elements in the layout.
+
+<DoDontRow>
+<DoDont
+colMd={4} colLg={4}
+caption="Do: use the default icon alignment to keep accordion text aligned with other content on the page.">
+
+![Do: use the default icon alignment to keep accordion text aligned with other content on the page.](./images/accordion-alignment-do.png)
+
+</DoDont>
+<DoDont
+colMd={4} colLg={4}
+type="dont"
+caption="Don’t: place icons on the left and create misalignment with the other text on the page.">
+
+![Don’t: place icons on the left and create misalignment with the other text on the page.](./images/accordion-alignment-dont.png)
+
+</DoDont>
+</DoDontRow>
+
+#### Modifier
+
+In rare scenarios, the accordion can be modified to place the icon in front of the title for only tree-like purposes. However, most instances should use the default alignment and for any pure content or documentation purposes it is required to use the default alignment. Icon placement should be consistent throughout your experience.
+
+### Placement
+
+Accordions can be placed with main page content or placed inside of a container such as a side panel or tile.
+
+<Row>
+<Column colLg={8}>
+
+![accordion placement](images/accordion-placement-2.png)
+
+</Column>
+</Row>
+
+#### Grid placement
```

Comment:
I think we still need to validate the grid placement stuff after our conversation last week. Is there a specific issue/list I can use to track which components we'd like to have work with the grid? 👀

Reply:
I thought we agreed to leave the content in because it demonstrates the correct and indented grid placement and we'd work out how to correctly document a hack on the code tab.  

Here is the issue for grid hanging components: https://github.com/carbon-design-system/carbon/issues/5892

Reply:
@aagonzales it seemed like the process we wanted to hit was one where design and development were coordinating around proposed functionality. It seems like we're at the phase of design proposing this functionality and I thought the idea was that dev would take that issue and figure out how feasible it would be before we would offer it as guidance. The question would be if we offer the guidance before we validate it then how do we handle what happens if we end up not being able to do it.

Thanks so much for the issue, will try and see if we can get this into our April work or at the very least make sure it's in May 🙏

Reply:
Yes, but I thought we agreed to document the design intent now as it can be achieved through other means outside of our grid package. 

That or we need change the component code, to have the rules and type alignment because we never want the accordion content to be indented from the other page content.

Reply:
@aagonzales I think we'd want to be careful around folks tackling this independently as it would require overriding styles on our components. This can cause unintended breakage down the line either due to unrelated updates for that component or updates to guidance. It'd be similar to hard coding a token instead of using a token.

Reply:
What is we talked about it from a type alignment perspective instead of grid placement. It is a fundamental design principle of the new language to have the strong vertical type alignment going down the page. What if we talked to that point instead of talking about grid gutters and hanging? I honestly don't care how it happens as long type inside the accordion and type on the page matches up.

Reply:
We just need to advise designers that this is how content should alignment when using accordion. 

![image](https://user-images.githubusercontent.com/11670886/79806917-2c68f900-832f-11ea-9c55-164e608c201d.png)

Reply:
^anything other than this alignment is unacceptable.

Reply:
That seems clear from a guidance perspective.

Unless guidance is impossible to implement, then we need to provide guidance on implementation - even (arguably, especially) - when it's unclear or hacky. The reality is, people already have to implement this guidance. We should offer a recommended or (if uncomfortable making a recommendation), a possible, implementation option.

The broader conversation of how to subsequently improve that experience (for this component, and for others) should fall outside the scope of this PR, I think.